reddit_liveupdate/templates/liveupdate.xml

```diff
@@ -1,12 +1,22 @@
 <%!
-    from r2.lib.filters import spaceCompress, safemarkdown
+    from r2.lib.filters import safemarkdown
     from r2.lib.template_helpers import get_domain
+    from r2.lib.template_helpers import rss_date, rss_author
 %>
-
-<item>
-    <guid isPermaLink="true">https://${get_domain(cname=False, subreddit=False)}/live/${c.liveupdate_event._id}/updates/${thing._id}</guid>
-    <pubDate>${thing._date.strftime('%a, %d %b %Y %H:%M:%S %z')}</pubDate>
-    <description>
-        ${spaceCompress(safemarkdown(thing.body))}
-    </description>
-</item>
+<%namespace name="utils" file="utils.xml"/>
+<entry>
+    %if not thing.author.deleted:
```

Comment:
It's a long constructed string literal. But it can be factored out above for sure